src/components/panel/panel.scss

```diff
@@ -182,5 +180,4 @@
   block
   p-2;
   width: fit-content;
-  z-index: 1;
```

Comment:
Are you sure this one isn't needed? If slotted user content has a z-index it might appear over this.

Reply:
I _think_ so. Currently, It looks like the [slotted content and the FAB containers don't overlap](https://codepen.io/jcfranco/pen/rNpMerg?editors=1010). Do you know if there's another configuration where they do?

Reply:
Yes, they can overlap like so:
https://codepen.io/driskull/pen/MWrjxZa?editors=1000

However, seems like a z-index messes it up currently anyway so might not be an issue.

In most cases they will overlap since a height is defined.

Reply:
I think these were here mainly to make sure that the slotted content is always below the header/fab/scrim/etc.

Reply:
Thanks for testing and the example. I suspected I was missing something.